README.md

```diff
@@ -174,8 +174,6 @@ Pros:
 Cons:
  - Not self-describing as to the effect on the `when` and the developer intent behind adding it
  - Poor locality as the property is far away from the `when` keyword it modifies
- - Impact on compiled code in the form of a property call, function call, and/or additional
-   instructions at the call-site
 
 ### Library trailing property
```

Comment:
Considering this option, maybe the previous one is explicitly and intentionally worse, to nudge readers towards a better more expressive solution?